### DISCUSSION TITLE
Issue with AHT20 sensor, provided AHTX_CMD  doesn't work for my sensor

### DISCUSSION THREAD

**Discussion by HenryJanssen** [arendst/Tasmota#20320] [Q&A]
I am using a combined AHT20 BMP 280 sensor. I succesfully built a binary, the BMP280 worked out of the box. The AHT 20 didn't work, comparing it with the microPython implementation I found that they were using a different value, I changed it accordingly:
 #define AHTX_CMD     0xBE // Cmd for AHT2x instead if **0xB1** in xsns_63_aht1x.ino
After this change it worked for me. Have other people built it succesfully with the setting as provided?

**Comment by sfromis** [comment]
Also discussed in https://github.com/arendst/Tasmota/discussions/20163
